readme.md

```diff
@@ -1,7 +1,10 @@
 
 ## Clear Linux OS Distribution Project
 
-This project is currently only used to track generic non-package
-specific issues.  We may close issues that are upstream package issues
-with the request that you file issues in that project. To file an
-issue, visit the github project page and click on 'issues'.
+This github project is used to track issues related to the Clear Linux OS Distribution.
+To elaborate, this project tracks issues related to the way Clearlinux packs software or requests about software that the ClearLinux OS should include
```

Comment:
Repeats the previous sentence almost entirely -- awkward?

Reply:
maybe just start with that second sentence, something like it.

and then list what possible outcomes there are for other issues.